spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/sql/CloudSqlEnvironmentPostProcessor.java

```diff
@@ -52,8 +53,7 @@
  * @author Eddú Meléndez
  */
 public class CloudSqlEnvironmentPostProcessor implements EnvironmentPostProcessor {
```

Comment:
Seeing now how handling both JDBC and R2DBC in one class here is not really reusing that much code, and making it less readable, maybe a separate `EventPostProcessor` is a better design after all. You can still re-use some code bt introducing a common `AbstractCloudSqlEnvironmentPostProcessor` that both would extend.